cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java

```diff
@@ -72,6 +72,12 @@ public void updateUser(final MetricUser user) {
     }
 
     @Override
+    public void tagUser(final String tagName, final MetricUser user) {
+        final User intercomUser = getIntercomUser(user);
+        Tag.tag(new Tag().setName(tagName), intercomUser);
```

Comment:
This line should be in a `try { ... } catch` block like the other methods in this class